README.rst

```diff
@@ -70,7 +70,7 @@ an older version of Celery:
 
 - Python 2.6: Celery series 3.1 or earlier.
 - Python 2.5: Celery series 3.0 or earlier.
-- Python 2.4 was Celery series 2.2 or earlier.
+- Python 2.4 was the Celery series 2.2 or earlier.
```

Comment:
I guess it could become consistent with the other two list items' format: `Python 2.4: ...`